src/main/java/org/smartregister/pathevaluator/ResourceType.java

```diff
@@ -16,7 +16,8 @@
 	PERSON("Person"),
 	TASK("Task"),
 	QUESTIONAIRRE_RESPONSE("QuestionnaireResponse"),
-	GLOBAL_TASK("Global.Task");
+	GLOBAL_TASK("Global.Task"),
+	BUNDLE("Bundle");
```

Comment:
Change this to Inventory or Stock

Reply:
This would impact on Plan Definition as well, it would also be changed a bit. @githengi

Reply:
Yes however we cannot use Bundle since its wrapper for resources of any type we can only use Device or DeviceBundle.

We may use Bundle for other use cases in future using Bundle without qualifying it would lead to collisions